lib/elixir/lib/exception.ex

```diff
@@ -1342,8 +1342,39 @@ defmodule UndefinedFunctionError do
       hint_for_loaded_module(module, function, arity, nil)
   end
 
-  defp hint(_module, _function, _arity, _loaded?) do
-    ""
+  defp hint(module, function, arity, _loaded?) do
+    normalized_module = normalized_module_name(module)
```

Comment:
I would call it `downcased`. Because it isn't really normalized (there is no such thing as a normal version).

Reply:
done